janitor/db/database.py

```diff
@@ -12,13 +12,14 @@
 
 
 class Database:
-    def __init__(self, creds: DbConnectionDetails):
+    def __init__(self, creds: DbConnectionDetails, autocommit=True):
```

Comment:
I have a feeling that using this means that every action on the database is its own transaction.  I may be wrong, but I think this undoes the efficiency you have in your `write_entries_to_table` method when trying to batch the writes together and commit at the end of the transaction.  It could be that I'm wrong and that the MySQL package is clever enough to only auto-commit when you haven't opened a transaction.

Reply:
That would make sense, but when I remove `self.connection.commit()` from the `write_entries_to_table` method, no entries get written to the table. So it seems like it is writing it all in one go at the end